app/code/Magento/SalesRule/Model/Rule/Condition/Product/Subselect.php

```diff
@@ -68,7 +68,13 @@ public function asXml($containerKey = 'conditions', $itemKey = 'condition')
      */
     public function loadAttributeOptions()
     {
-        $this->setAttributeOption(['qty' => __('total quantity'), 'base_row_total' => __('total amount')]);
+        $this->setAttributeOption(
+            [
+                'qty' => __('total quantity'),
+                'base_row_total' => __('total amount (excl. tax)'),
+                'row_total_incl_tax' => __('total amount (incl. tax)')
```

Comment:
I believe that we should use `base_row_total_incl_tax` here

Reply:
Hey @Den4ik thank you for the feedback. I've modified the suggested line.